src/main/java/seedu/module/model/ModuleManager.java

```diff
@@ -24,8 +24,10 @@
     private static HashMap<Module, Integer> moduleMediumWorkLoadDistribution;
     private static HashMap<Module, Integer> moduleHighWorkLoadDistribution;
     private static List<String> supportedModulesInStr;
-    private static final String[] arrOfModules = {"CS1101S", "CS1231S", "CS2030", "CS2040S", "CS2101",
-        "CS2103T", "CS2105", "CS2106", "CS3230", "CS3243", "CS3244", "IS1103", "ST2131"};
+    private static final String[] arrOfModules = {"CS1010S", "CS1101S", "CS1231S",
```

Comment:
I think the new modules should also be included in the UG table

Reply:
Yep included them. I went out for breakfast halfway lol